index.d.ts

```diff
@@ -2,8 +2,16 @@ declare namespace srcset {
 	interface SrcSetDefinition {
 		url: string;
 		width?: number;
+		/**
+		 * @deprecated height is no longer allowed in the srcset specification
+		 */
+		height?: number;
```

Comment:
I don't think we should add this. It was already absent, so TS users would not have used it.

Reply:
I kind of disagree with you here, but OTOH typescript has escape hatches if someone really wants to use `height`. I'll nix it.